PerfTools/AllocMonitor/scripts/edmModuleAllocMonitorAnalyze.py

```diff
@@ -963,10 +970,12 @@ def __init__(self,f, moduleCentric):
             if len(l) > 5 and l[0:2] == "#M":
                 (id,name,mType)=tuple(l[2:].split())
                 moduleNames[int(id)] = name
+                moduleTypes[int(id)] = mType
```

Comment:
How about instead of creating another list `moduleTypes` that is indexed in the same way as `moduleNames`
* Create a new class, e.g. `ModuleInfo` that holds both the name and type
* Have a single list `moduleInfos` instead of `moduleNames` and `moduleTypes`
  * Similarly `esModuleInfos` instead of `esModuleNames` and `esModuleTypes`
* Pass the `moduleInfos` through the functions instead of `moduleNames` and `moduleTypes` (same for `esModuleInfos`)
 
?